packit_service/worker/helpers/build/copr_build.py

```diff
@@ -535,6 +535,16 @@ def run_copr_build(self) -> TaskResults:
         """
         Run copr build using SRPM built by us.
         """
+        self._report(
+            state=BaseCommitStatus.error,
+            description="`srpm_build_deps` not defined.",
+            url="https://packit.dev/posts/copr-srpms/#deployment-phases",
+            check_names="srpm_build_deps-not-defined",
+            markdown_content="All SRPMs will be built in Copr since January 2023. "
+            "Please use `srpm_build_deps` to be sure that we don't break "
```

Comment:
could we link the `srpm_build_deps` in configuration docs here as well?

Reply:
added